### DISCUSSION TITLE
Can't get Kong Gateway working

### DISCUSSION THREAD

**Discussion by Entrebear** [Kong/kong#13547] [Help]
Hi

I used the install script to install Kong Gateway on a fresh installation of Ubuntu 22.04.

When I try to verify the install, I get an error that I am unable to connect to the port.

I have allowed the applicable ports in the ufw firewall.

Please can you assist.

**Comment by nowNick** [comment]
Hi @Entrebear - Could you share a link to the script you used?
Also it's highly possible that something is already running on the port that Kong is trying to use.

**Comment by Entrebear** [comment]
It was the script from the Installation page on the Kong site:
`bash <(curl -sS https://get.konghq.com/install) -p kong -v 3.7.1`
It appears as if the Kong service isn't starting at all. Not sure why, it was a installed on a stock standard, fresh install of Ubuntu.

**Reply by chobits** [reply]
Hi @Entrebear I think there are some messages the command left, could you show us the provided message after you ran the script, then we could look into it and see why it failed.
